euth/dashboard/templates/euth_dashboard/project_list.html

```diff
@@ -63,7 +63,22 @@ <h1 class="black-divider dashboard-title">{% trans 'Projects' %}</h1>
                             <a href="{% url 'project-detail' project.slug %}" class="btn btn-hover-primary btn-sm" title="{% trans 'view project' %}">
                                 <i class="fa fa-eye"></i>
                             </a>
-                        </span>
+                            {% if project.is_archivable %}
+                            <form action="{% url 'dashboard-project-archive' project.slug %}" method="post">
+                              {% csrf_token %}
+                              <button type="submit" class="btn btn-sm btn-hover-primary" title="{% trans 'archive project' %}">
+                                <i class="fa fa-archive"></i>
+                              </button>
+                            </form>
+                            {% elif project.is_archived %}
+                            <form action="{% url 'dashboard-project-unarchive' project.slug %}" method="post">
+                              {% csrf_token %}
+                              <button type="submit" class="btn btn-sm btn-hover-primary" title="{% trans 'unarchive project' %}">
+                                <i class="fa fa-archive"></i>
```

Comment:
You nee to hide this (and the button above) from screen readers (by setting aria-hidden='true'). The scree reader will than use the title attribute instead.